src/components/ChangelogDialog/components/Item/Item.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import {block} from '../../../utils/cn';
+import {Label} from '../../../Label';
+import {Button} from '../../../Button';
+import {Icon} from '../../../Icon';
+import {PlayIcon} from '../../../icons/PlayIcon';
+import {Picture} from '../Picture/Picture';
+import {ChangelogItem, OnStoryClick} from '../../types';
+import i18n from '../../i18n';
+
+import './Item.scss';
+
+const b = block('changelog-dialog-item');
+
+export interface ItemProps {
+    className?: string;
+    data: ChangelogItem;
+    onStoryClick?: OnStoryClick;
+}
+
+export function Item({className, data, onStoryClick}: ItemProps) {
+    return (
+        <div className={b(null, className)}>
+            <div className={b('meta')}>
+                <div className={b('date')}>{data.date}</div>
+                {data.isNew ? (
+                    <Label className={b('label-new')} theme="info">
+                        {i18n('label_new')}
+                    </Label>
+                ) : null}
+            </div>
+            <div className={b('content')}>
+                <div className={b('title')}>
+                    {data.title ?? `${i18n('label_version')}: ${data.version}`}
```

Comment:
I guess it would be more straightforward to require just `title`, without magically switching between `version` and `title`

Reply:
Done